go-controller/pkg/cluster/gateway_spare_intf.go

```diff
@@ -48,7 +49,7 @@ func initSpareGateway(nodeName string, clusterIPSubnet []string,
 	}
 
 	err = util.GatewayInit(clusterIPSubnet, nodeName, ifaceID, ipAddress,
-		macAddress, gwNextHop, subnet, false, 0, nodeportEnable)
+		macAddress, gwNextHop, subnet, false, 0)
```

Comment:
This 0 should be config.Gateway.VLANID

Reply:
@girishmg for "spare" I thought that was handled in the local vswitch (see a couple lines above), not in OVN itself?

Reply:
You are right. So, what you have is fine. Thanks.